### PR TITLE
Updating the plugin version to generate the new documentation

### PR DESCRIPTION
**Pull request by jespino**
The plugin version is relevant to generate the documentation of the server API,
was using the version 0.0.5 and we are now in the 0.0.13, so we need to have
that to include all the new API endpoints and hooks.

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Thank you, @jespino! Is there a way to manage this as part of monthly releases moving forward? We've encountered this issue before, and it would be ideal to proactively address it rather than encounter missing content on the dev site.

File: go.mod [hunk 1]
```diff
@@ -3,68 +3,48 @@ module mattermost-developer-documentation
 go 1.19
 
 require (
-	github.com/mattermost/mattermost/server/public v0.0.5
+	github.com/mattermost/mattermost/server/public v0.0.13
```
Comment (hanzei) [inline go.mod:6]: Nit: There is already a `v0.0.14`

**Review by hanzei — APPROVED** [review]
Thanks for keeping the docs up to date :+1:

File: go.mod (hunk 1 above)
Comment (jespino) [reply go.mod:6]: Let me update that.

**Comment by jespino** [comment]
@cwarnermm good point, I would love to have that some kind automatized, because if not is going to get again  out of date over a over again. Probably we can add something the CI. @saturninoabril do you know who would be the right person to add that CI check?

It would be something like checking if `go list -u -m github.com/mattermost/mattermost/server/public` output is contained in go.mod file.

**Review by agarciamontoro — APPROVED** [review]

**Comment by hanzei** [comment]
Alternatively, we can configure Dependabot to update `go.mod` automatically.

**Pull request merged**